boggle.js

```diff
@@ -29,37 +29,38 @@ function printBoggleBoard(boggleBoard) {
 //   console.log(allArrays);
 // }
 
-function randomRange(length) {
-  return Math.floor(Math.random() * length - 1);
+function randomRange(arrayLength) {
+  return Math.floor(Math.random() * arrayLength);
 }
 
-function removeElement(allArrays, index) {
-  let item = allArrays.splice(index, 1)[0];
+function removeElement(selectedDice, index) {
+  let item = selectedDice.splice(index, 1)[0];
```

Comment:
Because this function doesn't care that the argument represents dice, a name like `array` makes a bit more sense here. That is, `removeElement` as you've written it works with *any* array.